Logger/LogFactory.cs

```diff
@@ -1,10 +1,34 @@
-﻿namespace Logger;
-
-public class LogFactory
-{
-    public BaseLogger CreateLogger(string className)
-    {
-        
-        return null;
-    }
-}
+﻿using System;
+using System.Diagnostics;
+
+namespace Logger;
+
+public class LogFactory
+{
+    public string? LogFilePath { get; set; }
+    public void ConfigureLogger(string filePath)
+    {
+        LogFilePath = filePath;

```

Comment:
Try to keep your naming consistent for your variables.
```suggestion
        _FilePath = filePath;

```